src/@framework/strategies/jwt.strategy.ts

```diff
@@ -3,9 +3,9 @@ import { ConfigService } from '@nestjs/config';
 import { PassportStrategy } from '@nestjs/passport';
 import { InjectRepository } from '@nestjs/typeorm';
 import { ExtractJwt, Strategy } from 'passport-jwt';
-import { User } from 'src/user/entities';
+import { User } from 'src/modules/users/entities';
```

Comment:
replace all src with relative path to avoid some issues during testing

Reply:
done